src/Console/InstallsInertiaStacks.php

```diff
@@ -323,6 +323,176 @@ protected function installInertiaReactStack()
+                ->notName(['Welcome.js', 'Welcome.ts'])
+            );
+        }
+
+        // Tests...
+        if (! $this->installTests()) {
+            return 1;
+        }
+
+        if ($this->option('pest')) {
+            (new Filesystem)->copyDirectory(__DIR__.'/../../stubs/inertia-common/pest-tests/Feature', base_path('tests/Feature'));
+        } else {
+            (new Filesystem)->copyDirectory(__DIR__.'/../../stubs/inertia-common/tests/Feature', base_path('tests/Feature'));
+        }
+
+        // Routes...
+        copy(__DIR__.'/../../stubs/inertia-common/routes/web.php', base_path('routes/web.php'));
+        copy(__DIR__.'/../../stubs/inertia-common/routes/auth.php', base_path('routes/auth.php'));
+
+        // Tailwind / Vite...
+        copy(__DIR__.'/../../stubs/default/resources/css/app.css', resource_path('css/app.css'));
+        copy(__DIR__.'/../../stubs/default/postcss.config.js', base_path('postcss.config.js'));
+        copy(__DIR__.'/../../stubs/inertia-common/tailwind.config.js', base_path('tailwind.config.js'));
+        $prefix = '';
+        if ($this->option('typescript')) {
+            $prefix = '-ts';
+        }
+        if ($this->option('ssr')) {
+            copy(__DIR__.'/../../stubs/inertia-svelte'.$prefix.'/vite.config.ssr.js', base_path('vite.config.js'));
+        } else {
+            copy(__DIR__.'/../../stubs/inertia-svelte'.$prefix.'/vite.config.js', base_path('vite.config.js'));
+        }
+
+        if (file_exists(resource_path('js/app.js'))) {
+            unlink(resource_path('js/app.js'));
+        }
+
+        if ($this->option('typescript')) {
+            copy(__DIR__.'/../../stubs/inertia-svelte-ts/tsconfig.json', base_path('tsconfig.json'));
+            copy(__DIR__.'/../../stubs/inertia-svelte-ts/tsconfig.node.json', base_path('tsconfig.node.json'));
+            copy(__DIR__.'/../../stubs/inertia-svelte-ts/resources/js/app.ts', resource_path('js/app.ts'));
+            copy(__DIR__.'/../../stubs/inertia-svelte-ts/resources/js/route.ts', resource_path('js/route.ts'));
+
+            if (file_exists(resource_path('js/bootstrap.js'))) {
+                rename(resource_path('js/bootstrap.js'), resource_path('js/bootstrap.ts'));
+            }
+
+            $this->replaceInFile('vite build', 'php artisan ziggy:generate && tsc && vite build', base_path('package.json'));
+            $this->replaceInFile('.js', '.ts', base_path('vite.config.js'));
+            $this->replaceInFile('.js', '.ts', resource_path('views/app.blade.php'));
+            $this->replaceInFile('.vue', '.svelte', base_path('tailwind.config.js'));
+        } else {
+            copy(__DIR__.'/../../stubs/inertia-common/jsconfig.json', base_path('jsconfig.json'));
+            copy(__DIR__.'/../../stubs/inertia-svelte/resources/js/app.js', resource_path('js/app.js'));
+            copy(__DIR__.'/../../stubs/inertia-svelte/resources/js/route.js', resource_path('js/route.js'));
+
```

Comment:
is bootstrap.js missing here?

Reply:
no, the bootstrap file is created by laravel (`composer create-project laravel/laravel`) isn't it ?
the React/vue builder also doesn't implement the copy of bootstrap.js either : https://github.com/laravel/breeze/blob/7d66aa47283de1412af732a5d02dcaf54a600899/src/Console/InstallsInertiaStacks.php#L296

Reply:
true, I guess I have somehow deleted it, while trying out this pull request, because I manually copied the files.
Is there another way to install the breeze version from your branch directly? Something like composer require ... with the git branch?

Reply:
yes you can, check out the brach in your local, and rename the package name in composer to for `example my-laravel/breeze` after that add this code to your project composer (the laravel project you want to test on) : 
```
      ....
      "repositories": [
	    {
	        "type": "path",
	        "url": "../path-to/breeze-branch"
	    }
	],
       "require": {
          ...
      },
```
after that in your project you can run `composer require "my-laravel/breeze @dev"`
and you can continue with `php artisan breeze:install`.

Reply:
Thanks for the explanation and the Svelte support!

Reply:
You're welcome of course 😉, there are a lot of people among the svelte community who needs breeze with svelte support, but unfortunately the PR is closed for the moment.